app/lib/l10n/ods_flutter_en.arb

```diff
@@ -72,5 +108,147 @@
   "@componentButtonsDemoLow": {
     "description": "Component Buttons Demo Low emphasis",
     "type": "text"
+  },
+
+  "@_COMPONENTS_CARDS": {},
+  "componentCardsTitle": "Cards",
+  "@componentCardsTitle": {
+    "description": "Component Cards - Title",
+    "type": "text"
+  },
+  "componentCardsDescription": "Cards are important components that can be used to organise and present a number of different types of related information.",
+  "@componentCardsDescription": {
+    "description": "Component Cards - Description",
+    "type": "text"
+  },
+
+  "@_CARDS_VARIANT": {},
+  "cardSmallVariantTitle": "Card Small",
+  "@cardSmallVariantTitle": {
+    "description": "Component Card Small Screen - Title",
+    "type": "text"
+  },
+  "cardSmallVariantSubTitle": "OdsSmallCard",
+  "@cardSmallVariantSubTitle": {
```

Comment:
Same comment here: To rename in `cardSmallVariantSubtitle ` cause "subtitle" is one single word.